examples/sentiment_analysis_example.py

```diff
@@ -0,0 +1,90 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+from rev_ai import sentiment_analysis_client, apiclient
+from rev_ai.models import SentimentValue
+
+
+# String containing your access token
+access_token = "<your_access_token>"
+
+# Create your api client
+client = sentiment_analysis_client.SentimentAnalysisClient(access_token)
+
+# Submit a job with whatever text you want by changing this input
+text = "<input_text>"
```

Comment:
I think we should seed with some example text. The example should require as little input during the example run as possible.

Reply:
Seems reasonable. I'll use some text from wikipedia or something